pkg/controller/rollout/trafficrouting/trafficroutingutil.go

```diff
@@ -16,12 +16,17 @@ limitations under the License.
 
 package trafficrouting
 
+import "context"
```

Comment:
put interface in util file is strange, consider rename the file to interface.go or trafficrouting.go